global-api/importer/datasource_seeder/climatetrace_seeder.py

```diff
@@ -0,0 +1,341 @@
+"""seed datasource table with climatetrace data
+-- query to get start and end year
+SELECT
+DISTINCT
+    filename,
+    EXTRACT(YEAR FROM MIN(start_time)) AS start_year,
+    EXTRACT(YEAR FROM MAX(start_time)) AS end_year
+FROM "asset"
+GROUP BY "filename";
+"""
+import argparse
+import os
+from sqlalchemy import create_engine, MetaData, Table
+from utils import DataSource, uuid_generate_v3, insert_record
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--database_uri",
+        help="database URI (e.g. postgresql://ccglobal:@localhost/ccglobal)",
+        default=os.environ.get("DB_URI"),
+    )
+    args = parser.parse_args()
+
+    engine = create_engine(args.database_uri)
+    metadata_obj = MetaData()
+    table = Table("datasource", metadata_obj, autoload_with=engine)
+
+    BASE_URL = "https://ccglobal.openearth.dev"
```

Comment:
This needs to depend on the environment the global API is executed in later on. So depending on `ENV == "production"`, use the .cloud domain, otherwise .dev.

Reply:
Changed the code to this:

```python
    BASE_URL = "https://ccglobal.openearth.dev"

    if os.environ.get("ENV").lower() == 'production':
        BASE_URL = "https://ccglobal.openearth.cloud"
```